pom.xml

```diff
@@ -403,6 +418,22 @@
       <version>2.24.0</version>
       <scope>test</scope>
     </dependency>
+    <!-- Dependencies are required after updating bigquery.connector version from 0.13.1-hadoop2 to hadoop2-0.13.17 -->
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+      <version>1.4</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.flogger</groupId>
+      <artifactId>flogger-system-backend</artifactId>
+      <version>0.3.1</version>
```

Comment:
can you add properties for these versions

Reply:
Done.